File: Python/driver.py
class Driver:

    def __init__(self, driver_id, driver_name):
        self.driver_id = driver_id
        self.driver_name = driver_name

    def display_driver(self):
        print("---------------- Driver Details ------------------")
        print("Driver ID      :", self.driver_id)
        print("Driver Name    :", self.driver_name.capitalize())

File: Java/Driver.java
class Driver
{
    private int driverId;
    private String driverName;

    Driver(int driverId, String driverName)
    {
        this.driverId = driverId;
        this.driverName = driverName;
    }

    public void displayDriver()
    {
        System.out.println("---------------- Driver Details ------------------");
        System.out.println("Driver ID      : " + driverId);
        System.out.println("Driver Name    : " + capitalize(driverName));
    }

    public static String capitalize(String text)
    {
        if(text == null || text.isEmpty())
            return text;

        return text.substring(0,1).toUpperCase() +
               text.substring(1).toLowerCase();
    }
}